arch/risc-v/src/bl808/bl808_timerisr.c

```diff
@@ -101,6 +55,14 @@ static int bl808_ssoft_interrupt(int irq, void *context, void *arg)
 
 void up_timer_initialize(void)
 {
-  irq_attach(RISCV_IRQ_SSOFT, bl808_ssoft_interrupt, NULL);
-  up_enable_irq(RISCV_IRQ_SSOFT);
+  /* Initialize the OpenSBI Timer. mtime and mtimecmp are unused for
+   * OpenSBI.
+   */
+
+  struct oneshot_lowerhalf_s *lower = riscv_mtimer_initialize(
+    0, 0, RISCV_IRQ_STIMER, MTIMER_FREQ);
```

Comment:
Maybe better to move `riscv_mtimer_initialize` to the next line?

Reply:
Yep moved to the next line, thanks!